CHANGELOG.md

```diff
@@ -15,9 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Improve the device permission handling guide.
```

Comment:
nit: Improve device permissions handling in quickstarts?

Reply:
Updated.